Client/Frontend/Strings.swift

```diff
@@ -675,22 +707,28 @@ extension String {
 
         public struct FeltPrivacyUI {
             public static let Title = MZLocalizedString(
-                key: "FeltPrivacyUI.Title.v121",
+                key: "FirefoxHomepage.FeltPrivacyUI.Title.v122",
                 tableName: "FirefoxHomepage",
                 value: "Leave no traces on this device",
                 comment: "The title for the card that educates users about how private mode works. The card shows up on the homepage when in the new privacy mode.")
 
             public static let Body = MZLocalizedString(
-                key: "FeltPrivacyUI.Body.v121",
+                key: "FirefoxHomepage.FeltPrivacyUI.Body.v122",
                 tableName: "FirefoxHomepage",
                 value: "%@ deletes your cookies, history, and site data when you close all your private windows.",
                 comment: "The body of the message for the card that educates users about how private mode works. The card shows up on the homepage when in the new privacy mode. Placeholder refers to app name.")
 
             public static let Link = MZLocalizedString(
-                key: "FeltPrivacyUI.Link.v121",
+                key: "FirefoxHomepage.FeltPrivacyUI.Link.v122",
                 tableName: "FirefoxHomepage",
                 value: "Who might be able to see my activity?",
                 comment: "The link for the card that educates users about how private mode works. The link redirects to an external site for more information. The card shows up on the homepage when in the new privacy mode.")
+
+            public static let ToastTitle = MZLocalizedString(
+                key: "FirefoxHomepage.FeltPrivacyUI.Link.v122",
```

Comment:
do we want this be part of FeltDeletion instead of FeltPrivacyUI since is part of the deletion flow?

Reply:
oh I didn't realize we had to make a separate struct 😂 and the other one was under alert